.github/workflows/test-executor-emissary.yaml

```diff
@@ -0,0 +1,76 @@
+# This is an auto-generated file. DO NOT EDIT
```

Comment:
Add the source of these generated files?

Reply:
`gen-ci.sh`